crowdin.yaml.tmpl

```diff
@@ -0,0 +1,19 @@
+# Template for crowdin.com translation service
+project_identifier: django-formidable
+api_key: CROWDIN-API-KEY
```

Comment:
Too bad we can set this through a env var instead of re-generating the same file on every env.